server/conf/log4j2.properties.tmpl

```diff
@@ -1,4 +1,4 @@
-rootLogger.level=${sys:log4j.logLevel:-INFO}
+rootLogger.level={{ default .Env.SCALAR_DB_LOG_LEVEL "INFO" }}
```

Comment:
For the templating by the `dockerize` command, I updated this value according to the `dockerize` (Golang text/template) syntax.